samcli/commands/pipeline/bootstrap/pipeline_oidc_provider.py

```diff
@@ -0,0 +1,84 @@
+"""
+Represents a pipeline OIDC provider
+"""
+from abc import abstractmethod
+from typing import List
+import click
+
+from samcli.lib.config.samconfig import SamConfig
+
+
+class PipelineOidcProvider:
+
+    COMMON_OIDC_PARAMETERS_NAMES = ["--oidc-provider-url", "--oidc-client-id"]
+
+    def __init__(self, oidc_parameters: dict) -> None:
+        self.oidc_parameters = oidc_parameters
+
+    def verify_common_parameters(self) -> str:
+        error_string = ""
+        for parameter_name in self.COMMON_OIDC_PARAMETERS_NAMES:
+            if not self.oidc_parameters[parameter_name]:
+                error_string += f"Missing required parameter '{parameter_name}'\n"
+        return error_string
```

Comment:
`verify_common_parameters` and `verify_subject_claim_parameters` seems share the same logic. Can we create a method to be shared between these classes so we can avoid some duplicates?